### PR TITLE
MessageComponentInterface without prior use statement

### PR DESCRIPTION
**Pull request by dozymoe**
There is silent error in App->route() about using controller that implements MessageComponentInterface

### CODE REVIEW

**Comment by dozymoe** [comment]
Ah, sorry, didn't notice there are two MessageComponentInterface.

**Comment by dozymoe** [comment]
Nevermind, those interfaces have different parameters for onMessage method.

**Comment by cboden** [comment]
Sorry for the confusion. Naming things is hard. 🤔

**Comment by dozymoe** [comment]
The patch is still applicable though, I was hasty.
They were both will be thrown into WsServer that can accept both.
The second argument in Ratchet\MessageComponentInterface is a string, while the one in Ratchet\WebSocket\MessageComponentInterface is Ratchet\RFC6455\Messaging\MessageInterface in case that was useful to some.

**Comment by matheusb-comp** [comment]
This change solves #786

File: src/Ratchet/App.php [hunk 1]
```diff
@@ -11,6 +11,7 @@
 use Ratchet\Server\FlashPolicy;
 use Ratchet\Http\HttpServer;
 use Ratchet\Http\Router;
+use Ratchet\WebSocket\MessageComponentInterface as DataComponentInterface;
```
Comment (cboden) [inline src/Ratchet/App.php]: I think `as WsComponentInterface` would be a better name here. There is another interface in the RFC lib named `DataInterfae` which I wouldn't want to be conflated with here.

**Comment by cboden** [comment]
The PR looks good if you want to re-open it and make that one small naming convention change. 

I see the underlying problem. `\Ratchet\WebSocket\MessageComponentInterface` used to extend `\Ratchet\MessageComponentInterface` but that was broken up into smaller and different interfaces in order to support binary messages. I forgot to update App accordingly.

**Pull request closed without merging**